tools/deb/istio-iptables.sh

```diff
@@ -143,6 +150,13 @@ if [ -n "${ISTIO_INBOUND_PORTS:-}" ]; then
           iptables -t nat -A ISTIO_INBOUND -p tcp --dport ${port} -j ISTIO_REDIRECT
       done
   fi
+else
+  # ISTIO_INBOUND_PORTS not set, but script called in k8s backward compatibility mode via args
+  if [ "${K8S_MODE}" == "1" ]; then
+    iptables -t nat -N ISTIO_INBOUND
+    iptables -t nat -A PREROUTING -p tcp -j ISTIO_INBOUND
+    iptables -t nat -A ISTIO_INBOUND -p tcp -j ISTIO_REDIRECT
+  fi
```

Comment:
else echo warning something is off?

Reply:
Nothing off - if running in K8S_MODE (i.e. in 0.3, when we use the new script in both
k8s and raw vm), the default will remain unchanged in k8s, for backward compat.

In rawVM we want the default to be to intercept only what is explicitly configured since VMs are more likely to run other services. But there is nothing to warn or wrong.